packages/theming/src/index.ts

```diff
@@ -24,7 +24,7 @@ export { useWindow } from './utils/useWindow';
 export { useText } from './utils/useText';
 export { default as menuStyles } from './utils/menuStyles';
 export { focusStyles, SELECTOR_FOCUS_VISIBLE } from './utils/focusStyles';
-export { StyledBaseIcon } from './utils/StyledBaseIcon';
+export { StyledBaseIcon, type IStyledBaseIconProps } from './utils/StyledBaseIcon';
```

Comment:
I'm ok with exporting this, but:
1) I wouldn't expect us to reuse within other packages; in other words, it's for consumers-only
2) it should be moved to the `type` export section below